sdk/storage/azure-storage-blob/README.md

```diff
@@ -99,20 +110,20 @@ use of a dedicated client object.
 
 ### Clients
 Four different clients are provided to to interact with the various components of the Blob Service:
-1. **[BlobServiceClient](https://docs.microsoft.com/en-us/python/api/azure-storage-blob/azure.storage.blob.blobserviceclient)** -
+1. **[BlobServiceClient](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-storage-blob/12.0.0b5/azure.storage.blob.html#azure.storage.blob.BlobServiceClient)** -
```

Comment:
Same comments I had above - I thought the final docs URLs were supposed to be the docs.microsoft.com ones - but I could be wrong? 
Also, final version will be 12.0.0